app/controllers/todos_controller.rb

```diff
@@ -1,7 +1,21 @@
 class TodosController < ApplicationController
+    @@sort_Type = "none"
```

Comment:
Are we sure we want a class variable instead of an instance variable?

Reply:
You are correct, when doing pagination I realised how I can do this as an instance variable and that will be changed. Thank you.